src/utils/valideringUtils.ts

```diff
@@ -21,8 +23,9 @@ interface Begrunnelser {
 const MAX_LENGTH_STED = 200;
 
 export const texts = {
-  dateMissing: "Vennligst angi dato",
-  dateWrongFormat: "Vennligst angi riktig datoformat; dd.mm.åååå",
+  dateMissing: "Vennligst angi gyldig dato",
```

Comment:
Den nye DatePicker-komponenten gir verdi `undefined` dersom brukeren skriver inn en dato som er "disabled" (utenfor lovlig min og max-verdi). Ref tråd her: https://nav-it.slack.com/archives/C7NE7A8UF/p1679585187344759. Endret derfor litt på denne slik at den gir mening både for tom dato og ugyldig dato.